src/components/Menu/Menu.tsx

```diff
@@ -13,23 +14,21 @@ import "./Menu.css";
 
 export type MenuProps = Omit<ReactMenuProps, "menuButton" | "theming" | "transition"> & {
   isDisabled?: boolean;
+  menuButton?: ReactMenuProps["menuButton"];
 };
 
 const Menu: React.FC<MenuProps> = (props) => {
   const { colorMode } = useColorMode();
+  const menuButton = props.menuButton ?? (
+    <IconButton aria-label="Menu" icon={<TbDots />} variant="ghost" isDisabled={props.isDisabled} />
+  );
+
   return (
-    <Box>
+    <Box zIndex={theme.zIndices.dropdown}>
       <ReactMenu
         align={"end"}
```

Comment:
```suggestion
        {...props}
        align={"end"}
```

Should we also pass down the rest of the props?

Also for some reason, `align="end"` has stopped working
<img width="369" alt="image" src="https://github.com/tarasglek/chatcraft.org/assets/78865303/54b78af1-b98e-4605-a179-4aa543680465">


In Prod:
<img width="483" alt="image" src="https://github.com/tarasglek/chatcraft.org/assets/78865303/bed7f2ac-f967-4986-82ef-7984f5452065">

Reply:
What is `align="end"` supposed to be doing here?  Is it what is breaking the width?  I'm not sure what the right fix is for that, could use some help.

Reply:
`align` is used to position the menu/submenu contents with respect to the **anchor** element.
![image](https://github.com/tarasglek/chatcraft.org/assets/78865303/ac894940-b7a3-4c2a-9331-0f028ec49cac)

In this case, it should be "start" so the contents do not touch the edge. (I was mistaken that it should be "end")
Also, instead of overriding the default `align` value here (which is "start"), we can change it wherever we need a different value.
For example, in `MessageBase` we need it to be `end` so we do not have a horizontal scrollbar.
```tsx
<Menu align="end" isDisabled={isLoading}>
  <MenuItem onClick={handleCopy} icon={<MdContentCopy />}>
    Copy
  </MenuItem>
```



For the **width**, I did not find any official property in the docs. But we can use `menuStyle` to set a `minWidth`.
```tsx
// Props that we want to force-override can go after `{...props}`
return (
  <Box zIndex={theme.zIndices.dropdown}>
    <ReactMenu
      transition={true}
      theming={colorMode === "dark" ? "dark" : undefined}
      {...props}
      menuButton={menuButton}
      menuStyle={{
        minWidth: "225px",
        ...props.menuStyle,
      }}
    >
      {props.children}
    </ReactMenu>
  </Box>
);
```

**Final Result:**
![image](https://github.com/tarasglek/chatcraft.org/assets/78865303/37a89c13-380f-40d7-81d8-317261fdcf80)

However, even though all of this works, I don't like how we have to micro-manage the positioning of the menu in different scenarios. Not sure, if its the component's design or we are using it wrong.